connector/queue/model.py

```diff
@@ -76,6 +76,7 @@ class QueueJob(models.Model):
     date_enqueued = fields.Datetime(string='Enqueue Time', readonly=True)
     date_done = fields.Datetime(string='Date Done', readonly=True)
     eta = fields.Datetime(string='Execute only after')
+    retry_pattern = fields.Char(string='Retry Pattern')
```

Comment:
I forgot to remove this field. Could remove it or merge https://github.com/apertoso/connector/pull/2?